cluster-up/cluster/kind/common.sh

```diff
@@ -172,8 +156,10 @@ function setup_kind() {
     docker cp ${CLUSTER_NAME}-control-plane:/kind/bin/kubectl ${KUBEVIRTCI_CONFIG_PATH}/$KUBEVIRT_PROVIDER/.kubectl
     chmod u+x ${KUBEVIRTCI_CONFIG_PATH}/$KUBEVIRT_PROVIDER/.kubectl
 
+    mkdir -p ${KUBEVIRTCI_CONFIG_PATH}/$KUBEVIRT_PROVIDER/cni-plugins
+    curl -L https://github.com/containernetworking/plugins/releases/download/v0.8.5/cni-plugins-linux-amd64-v0.8.5.tgz | tar xz -C ${KUBEVIRTCI_CONFIG_PATH}/$KUBEVIRT_PROVIDER/cni-plugins
     for node in $(_get_nodes | awk '{print $1}'); do
-        docker exec $node /bin/sh -c "curl -L https://github.com/containernetworking/plugins/releases/download/v0.8.5/cni-plugins-linux-amd64-v0.8.5.tgz | tar xz -C /opt/cni/bin"
+        docker cp ${KUBEVIRTCI_CONFIG_PATH}/$KUBEVIRT_PROVIDER/cni-plugins/. $node:/opt/cni/bin
     done
```

Comment:
This is not related and leaves a tarball and a directory there, also the number of nodes we usually have is quite small so is like 2 curl + unpacks vs 1 curl + unpack.

Reply:
My intent was to reduce as much time possible for cluster-up to finish so I prefer to curl and unpack one time
and only then copy the content
I can add remove the tar and the directory right after the for loop, is that ok with you?

Reply:
Well but this is really not related to run etcd in memory, we can do that at different PR.

Reply:
No problem I will remove this commit